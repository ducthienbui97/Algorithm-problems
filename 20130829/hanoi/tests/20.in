3
20
BC BA CA CB AB AC
20
AC AB BC BA CB CA
20
BA AB BC CB AC CA
