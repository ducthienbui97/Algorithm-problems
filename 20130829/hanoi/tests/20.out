1048575
1162261467
2324522933
